### PR TITLE
add code of conduct revision"

### PR DESCRIPTION
**Pull request by ericholscher**
As noted in Trello, this is drawn almost verbatim from the Django CoC. See text inline for further attributions.
This PR is so folks can comment on the approach. It's not ready for merge, because we need at least a revised response/enforcement doc/page to go with it. I'll work on it next, after we agree on the CoC itself.
Once we have the CoC and enforcement updated, we can produce the "Welcome Wagon" content -- some revisions to the current Slack page, plus a Slack welcome message and maybe some other goodies.

Reopen of #115

### CODE REVIEW

**Comment by mjang** [comment]
FYI, when we complete this, I'll start a PR for a one sheet bit appropriate for Meetup announcements. Based on the current draft, I'd add bullet points based on the current highlights:

* Be friendly and patient
* Be welcoming
* Be (etc....)

Include a placeholder for organizer names / email addresses

(Partially inspired by Rose's comment in the meetups channel from the Boston CSS meetup)

File: docs/code-of-conduct.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Code of Conduct
```
Comment (mjang) [inline docs/code-of-conduct.rst:1]: Please see https://github.com/writethedocs/www/pull/115/commits/ef1be4587e35ad99a53575591c92ef8508ad1142 for comments

**Comment by mjang** [comment]
Also see https://github.com/nbkaplan's PR 119 (https://github.com/writethedocs/www/pull/119)

**Comment by ericholscher** [comment]
Closing in deference to Jennifers latest draft.

**Pull request closed without merging**